dingo/gw/waveform_generator/waveform_generator.py

```diff
@@ -16,7 +16,7 @@
     from lalsimulation.gwsignal.models import (
         gwsignal_get_waveform_generator as new_interface_get_waveform_generator,
     )
-except ImportError:
+except:
```

Comment:
This should raise an ImportError, no? A generic exception is not good practice.
```suggestion
except ImportError:
```